packages/myst-spec-ext/src/types.ts

```diff
@@ -64,6 +64,16 @@ export type Image = SpecImage & {
   height?: string;
 };
 
+export type Iframe = {
+  type: 'iframe';
+  label?: string;
+  identifier?: string;
+  src?: string;
```

Comment:
This is functionally required, I think - but maybe there's a reason to not be required in the type.

Reply:
Oh, yes, good point!

Reply:
```suggestion
  src: string;
```